Representable/representable
# add csv export for individual communities

**changes**
- as per partner request, added ability to export a community as a csv
- preserves coordinates and community info
**to test**
- go to community submission page
- click "export as CSV"
- check that CSV looks as expected
**screenshots**
![image](https://user-images.githubusercontent.com/41943646/102668737-c9836800-4130-11eb-916f-c3d1da332c0c.png)

**todo**
- csv will be most useful for multi-export, so we should add that to the roadmap!
